meinberlin/apps/budgeting/assets/SupportBox.jsx

```diff
@@ -63,7 +67,11 @@ export const SupportBox = (props) => {
       })
   }
 
-  const handleSupport = () => {
+  const handleSupport = (e) => {
+    if (props.isReadOnly) {
+      e.preventDefault()
```

Comment:
So now the button does also not react when not logged in and the isReadOnly is true. Is that intended or should we maybe change the order of the if statements, so if not logged in, it always redirects to the log in?

Reply:
But yeah, on the other hand it is annoying if you then log in and cant do anything anyway afterwards..

Reply:
I feel this is more consistent behavior and would leave it up to Marcelo to say what is better for the user